modules/csharp_api/csharp/CSharpAPI.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;net6.0;net48</TargetFrameworks>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>    
+    <PackageId>OpenVINO.CSharp.win</PackageId>
```

Comment:
Please, rename package from `OpenVINO.CSharp.win` -> `OpenVINO.CSharp` since it do not have any Windows specific API call